### PR TITLE
Update Dino Pirannah to LiveActor

### PR DESCRIPTION
**Pull request by cbondurant**
This moves the DinoPirannah actor to the new LiveActor system and resolves a missing PartsModel for the tail.

### CODE REVIEW

File: src/SuperMarioGalaxy/Actors/MiscActor.ts [hunk 1]
```diff
@@ -9985,3 +9985,25 @@ export class HomingKillerLauncher extends LiveActor<HomingKillerLauncherNrv> {
         }
     }
 }
+
+export class DinoPackun extends LiveActor {
+
+    public tail: PartsModel|null;
```
Comment (magcius) [inline src/SuperMarioGalaxy/Actors/MiscActor.ts:9991]: It should always exist, so remove the | null.

Comment (magcius) [inline src/SuperMarioGalaxy/Actors/MiscActor.ts]: Extra blank line as well

File: src/SuperMarioGalaxy/Actors/MiscActor.ts [hunk 2]
```diff
@@ -9985,3 +9985,25 @@ export class HomingKillerLauncher extends LiveActor<HomingKillerLauncherNrv> {
...
+    constructor(zoneAndLayer: ZoneAndLayer, sceneObjHolder: SceneObjHolder, infoIter: JMapInfoIter) {
+        super(zoneAndLayer, sceneObjHolder, 'DinoPackun');
+        this.initModelManagerWithAnm(sceneObjHolder, "DinoPackun");
+        connectToSceneEnemy(sceneObjHolder, this);
+        initLightCtrl(sceneObjHolder, this);
+        this.initBinder(150, 150,0);
+        this.initEffectKeeper(sceneObjHolder, null);
+        initDefaultPos(sceneObjHolder, this, infoIter);
+        this.tail = createPartsModelEnemyAndFix(sceneObjHolder,this, "DinoPackunTailBall",null,null,null, "Tail7");
+        startBck(this,"wait");
```
Comment (magcius) [inline src/SuperMarioGalaxy/Actors/MiscActor.ts:10002]: There's a few whitespace issues with this -- please make sure you have spaces after commas.

File: src/SuperMarioGalaxy/NameObjFactory.ts [hunk 3]
```diff
@@ -111,6 +111,9 @@ const ActorTable: NameObjFactoryTableEntry[] = [
     _("HomingKiller",                   HomingKillerLauncher),
     _("Torpedo",                        HomingKillerLauncher),
     _("MagnumKiller",                   HomingKillerLauncher),
+    _("DinoPackun",                     DinoPackun),
```
Comment (magcius) [inline src/SuperMarioGalaxy/NameObjFactory.ts:114]: The table in the game appears to only have "DinoPackun" and "DinoPackunVs2". Whatever I was doing before in the legacy actor switch statement was a mistake.

File: src/SuperMarioGalaxy/Actors/MiscActor.ts (hunk 1 above)
Comment (cbondurant) [reply src/SuperMarioGalaxy/Actors/MiscActor.ts]: woops, thanks

**Pull request merged**